src/pages/profile.js

```diff
@@ -0,0 +1,177 @@
+import Avatar from "../assets/avatar-profile.png";
+import NavBar from "../components/navBar";
+import Pencil from "../assets/pencil-icon.png";
+import Suitcase from "../assets/role_icon.png";
+import Pin from "../assets/location_icon.png";
+import Discord from "../assets/discord_icon.png";
+import Envelope from "../assets/email_icon.png";
+import InvisibleLab from "../assets/invisible_lab_icon.png";
+import Onda from "../assets/onda_logo.png";
+import Sub from "../assets/sub_logo.png";
+import PinkRoom from "../assets/pink_room_logo.png";
+import Universe from "../assets/universe.jpg";
+
+import List from "../components/list";
+import Info from "../components/info";
+import { GET } from "../utils/fetch";
+import { useState, useEffect } from "react";
+
+const current_user_id = 6; //localStorage.getItem("current");
+
+const companyLogos = {
+  Subvisual: Sub,
+  "Invisible lab": InvisibleLab,
+  Onda: Onda,
+  "Pink Room": PinkRoom,
+  Universe: Universe,
+};
+
+const formatDate = (dateString) => {
+  const date = new Date(dateString);
+  const options = { month: "short", day: "2-digit" };
+  return date.toLocaleString("en-US", options);
+};
+
+const getAnswers = async () => {
+  var response = await GET("users/" + current_user_id + "/answers");
+  return response;
+};
+
+const getSkills = async () => {
+  var response = await GET("users/" + current_user_id + "/skills");
+  return response.map((skill) => skill["name"]);
+};
+
+const getHobbies = async () => {
+  var response = await GET("users/" + current_user_id + "/hobbies");
+  return response.map((hobby) => hobby["name"]);
+};
+
+const getUserData = async () => {
+  var userData = await GET("users/" + current_user_id);
+  var company = await GET("/companies/" + userData.company_id.toString());
+  return {
+    userData: userData,
+    company: company,
+  };
+};
```

Comment:
This logic should be isolated. You can create a custom hook and have all the requests there, then when a component needs it, you just need call from hook